docs/cli-tools.rst

```diff
@@ -49,6 +49,100 @@ gt_mpi_gather
 
     gt_mpi_gather -j <query.json>
 
+  Here is an example `query.json` file:
+
+  .. code-block:: python
+
+    {
+        "workspace" : "/tmp/ws/",
+        "array" : "t0_1_2",
+        "query_column_ranges" : [ [ [0, 100 ], 500 ] ],
+        "query_row_ranges" : [ [ [0, 2 ] ] ],
+        "vid_mapping_file": "tests/inputs/vid.json",
+        "callset_mapping_file": "tests/inputs/callset_mapping.json",
+        "query_attributes" : [ "REF", "ALT", "BaseQRankSum", "MQ", "MQ0", "ClippingRankSum", "MQRankSum", "ReadPosRankSum", "DP", "GT", "GQ", "SB", "AD", "PL", "DP_FORMAT", "MIN_DP" ]
+    }
+
+  Most of the fields are self-explanatory (take a look at the :ref:`terminology section <Concepts + Terminology>` section).
+  The following fields are mandatory:
+  
+  * workspace (type: string or list of strings)
+  * array (type: string or list of strings)
+  * query_column_ranges : This field contains is a list of lists. Each member list contains the column ranges that are to be queried. Each element of the inner list can be a single integer, representing the single column position to be queried, or a list of size 2 representing the column range to be queried.
+
+    In the above example, the process will query column range \[0-100\] (inclusive) and the position 500 and return all *VariantCalls* intersecting with these query ranges/positions.
+    This field is a list of lists (and other fields are lists of strings), in order to support using GenomicsDB with MPI. Each member of the othermost list can be processed by a separate MPI rank.
```

Comment:
It looks like this paragraph was reworded to remove the reference to the wiki, but could we just drop it entirely? I may be missing a larger context, but at least in my mind this seems more like an implementation detail than something a docs consumer would need to know.

Reply:
Good point that I don't explicitly call out why this is really relevant to the user. The thing that is relevant for a docs consumer/user is that unless MPI is being used, only the first element of the outermost list is processed. I'll add wording to that effect.